lightning/src/ln/channelmanager.rs

```diff
@@ -7772,6 +7772,8 @@ where
 		let invoice_request = builder.build_and_sign()?;
 		let reply_path = self.create_blinded_path().map_err(|_| Bolt12SemanticError::MissingPaths)?;
 
+		let _persistence_guard = PersistenceNotifierGuard::notify_on_drop(self);
```

Comment:
Is this also needed in `create_refund_builder` where we create an outbound payment awaiting an invoice?